integration/mediation-tests/tests-sample/src/test/java/org/wso2/carbon/esb/samples/test/transport/Sample254TestCase.java

```diff
@@ -64,6 +64,16 @@ public void setEnvironment() throws Exception {
         FileUtils.deleteDirectory(inFolder);
         FileUtils.deleteDirectory(originalFolder);
 
+        while (outFolder.exists()) {
```

Comment:
We should limit this loop to some number of iterations. Otherwise the test will never return in an error situation.